src/cmd/dut.rs

```diff
@@ -142,16 +146,16 @@ fn run_dut_vnc(args: &ArgsVnc) -> Result<()> {
     let web_port = args.vnc_port.unwrap_or(6080);
 
     if let Err(e) = target.run_cmd_piped(&["kill -9 $(pgrep --full vnc)"]) {
-        eprintln!("Failed to kill previous vnc instance: {e}")
+        error!("Failed to kill previous vnc instance: {e}")
     }
 
     let mut child_kmsvnc = target.start_port_forwarding(vnc_port, 5900, "kmsvnc")?;
     let mut child_novnc = target.start_port_forwarding(web_port, 6080, "novnc")?;
 
-    eprintln!("To use VNC via web browser, please open:");
-    eprintln!("  http://localhost:{web_port}/vnc.html");
-    eprintln!("To connect via VNC client directly, use:");
-    eprintln!("  xtightvncviewer -encodings raw localhost:{vnc_port}");
+    warn!("To use VNC via web browser, please open:");
+    warn!("  http://localhost:{web_port}/vnc.html");
+    warn!("To connect via VNC client directly, use:");
+    warn!("  xtightvncviewer -encodings raw localhost:{vnc_port}");
```

Comment:
This is not a warning message but an information for user to tell how to open VNC.

Reply:
I think it can be warn to ensure the message is printed out to the users.